app/javascript/mastodon/locales/defaultMessages.json

```diff
@@ -3236,7 +3240,7 @@
         "id": "onboarding.steps.follow_people.title"
       },
       {
-        "defaultMessage": "You curate your own feed. Let's fill it with interesting people.",
+        "defaultMessage": "You curate your own feed. Lets fill it with interesting people.",
```

Comment:
This has wrongfully snuck in during a rebase and should be fixed!

Reply:
> 
Hey, @ClearlyClaire, thanks for the review!

I used "refresh feed/timeline" in most of the implementation due to the issue's request and saw that a little blue bird also calls this feature in a similar fashion, but after opening the PR and  @tribela's comment, I figured the best solution would be to focus on the first element in a `scrollable` section.

After analyzing the codebase more thoroughly, I realized that there is already an implementation that does this called `focusColumn` that utilizes the `'1', '2', '3', '4', '5', '6', '7', '8', '9'` keys, but this is only true if the advanced web interface option is enabled.

Maybe my solution would make more sense if `refresh timeline` and `refresh feed` were called something else. Any suggestions on potential names to refactor?